tests/arch/arm/arm_irq_vector_table/src/arm_irq_vector_table.c

```diff
@@ -188,33 +179,35 @@ void nrfx_power_clock_irq_handler(void);
 #if defined(CONFIG_BOARD_QEMU_CORTEX_M0)
 void timer0_nrf_isr(void);
 vth __irq_vector_table _irq_vector_table[] = {
-	nrfx_power_clock_irq_handler, 0, 0, 0, 0, 0, 0, 0,
-	timer0_nrf_isr, isr0, isr1, isr2
+	[POWER_CLOCK_IRQn]nrfx_power_clock_irq_handler,
+	[TIMER0_IRQn]timer0_nrf_isr,
+	[_ISR_OFFSET]isr0, isr1, isr2
 };
 #else
 vth __irq_vector_table _irq_vector_table[] = {
-	nrfx_power_clock_irq_handler,
-	isr0, isr1, isr2,
-	0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-	rtc_nrf_isr
+	[POWER_CLOCK_IRQn]nrfx_power_clock_irq_handler,
+	[_ISR_OFFSET]isr0, isr1, isr2,
+	[RTC1_IRQn]rtc_nrf_isr,
+	[ZEPHYR_CONSOLE_IRQ_NUM]CONSOLE_UART_IRQ_HANDLER,
```

Comment:
It should not be needed to install UART interrupt for this test. The related changes should be dropped.